MVP/defaults-build.yaml

```diff
@@ -26,6 +26,8 @@
         - timestamps
         - build-name:
             name: ${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}
+        - inject:
+            script-content: echo "create_time=$(date --utc +%FT%TZ)" >> $WORKSPACE/job.properties
```

Comment:
We have to change ">" to ">>" in sample_job here, anyway.
But yes, it would be an issue if existing jobs only pull and apply default-build. We probably need to figure out a solution later.